httpcore5/src/main/java/org/apache/hc/core5/http/io/entity/SerializableEntity.java

```diff
@@ -91,33 +100,33 @@ private void createBytes(final Serializable ser) throws IOException {
     }
 
     @Override
-    public InputStream getContent() throws IOException, IllegalStateException {
+    public final InputStream getContent() throws IOException, IllegalStateException {
         if (this.objSer == null) {
             createBytes(this.objRef);
```

Comment:
Based on this (apparent) lazy initialization path, this class is not currently immutable in any sense.

Reply:
@rschmitt I removed `IMMUTABLE_CONDITIONAL` annotation from `ByteBufferEntity` and `InputStreamrEntity` and removed buffering code from `SerializableEntity`.